### PR TITLE
fix:Add Anjuli08

### LINKED ISSUE #1478: Anjuli Aggarwal
## Profile Screenshot

Screenshot (81)

### PR DESCRIPTION
**Pull request by Anjuli08**
resolves #1478 

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

### CODE REVIEW

File: profiles/Anjuli08.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/90701800/194690166-9327c711-0e96-4420-a1db-efbbe7807147.png",
+    "issueId": 1478,
+    "name": "Anjuli Aggarwal",
+    "username": "Anjuli08"
+}
```
Comment (Panquesito7) [inline profiles/Anjuli08.json:6]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by Anjuli08** [comment]
done! pls review

File: profiles/Anjuli08.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
```
Comment (Panquesito7) [inline profiles/Anjuli08.json:7]:
```suggestion
```

File: profiles/Anjuli08.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/90701800/194690166-9327c711-0e96-4420-a1db-efbbe7807147.png",
+    "issueId": 1478,
+    "name": "Anjuli Aggarwal",
+    "username": "Anjuli08"
```
Comment (Panquesito7) [inline profiles/Anjuli08.json:5]: Should be a two-space indentation.
```suggestion
  "image": "https://user-images.githubusercontent.com/90701800/194690166-9327c711-0e96-4420-a1db-efbbe7807147.png",
  "issueId": 1478,
  "name": "Anjuli Aggarwal",
  "username": "Anjuli08"
```

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Comment by Anjuli08** [comment]
Thank you!

**Pull request merged**